### DISCUSSION TITLE
Show quantity/count of associated recipes when browsing Tags/Categories

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1402] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
When browsing by Tags/Categories, it's helpful to know if you're going to enter into one that has only a few, or a lot of associated recipes. There's currently no way to determine this before going into each. While managing, this can also help determine if some categories/tags are unused or underused and can be removed.

**(Optional): Suggest A Solution**
It'd be great if each Tag/Category label (under the Tags/Categories tabs) showed a count on the right side of the label of how many recipes are associated, and will be revealed once clicked.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by mechanarchy** [comment]
I second this, particularly
> It'd be great if each Tag/Category label (under the Tags/Categories tabs) showed a count on the right side of the label of how many recipes are associated, and will be revealed once clicked.

I'm currently trying to align and simplify the 'Labels' in my installation, and the Data Management table for this type is very sparse:
<img width="1197" alt="image" src="https://github.com/user-attachments/assets/f1561ef6-8635-4693-9658-05fbc9fa898d" />

The only columns available here are Id and Name; I feel this would really be benefited by another column showing the number of 'Foods' with this label attached. Even better would be a 🔍  icon in each row, letting you immediately jump to the 'Foods' table filtered to show just Foods with the given label applied.

Regretfully I have no time to develop a PR for this suggestion but I feel like it shouldn't be a particularly onerous feature to implement.

**Reply by mechanarchy** [reply]
Add a new UI column definition here:
https://github.com/mealie-recipes/mealie/blob/cb9008bb5c165409861b3e5762b62393f99f569b/frontend/pages/group/data/labels.vue#L160-L171

Server `GET` model definition:
https://github.com/mealie-recipes/mealie/blob/cb9008bb5c165409861b3e5762b62393f99f569b/mealie/schema/labels/multi_purpose_label.py#L33-L34
